async-nats/src/lib.rs

```diff
@@ -250,6 +250,10 @@ pub enum Command {
     },
     TryFlush,
     Connect(ConnectInfo),
+    GetDroppedMessages {
```

Comment:
Not to keep on going two sends for what could be an atomic add.
Want to split this out? we can merge the connection side things without controversy.